src/elm/Main/Pages/Model.elm

```diff
@@ -40,6 +40,8 @@ import Pages.Org_.Repo_.Tags
 import View exposing (View)
 
 
+{-| Model : The main model for the application.
+-}
```

Comment:
i think this will get overwritten by `elm-land generate`, should we remove it?

Reply:
removed